backend/src/nodes/nodes/pytorch/upscale_face.py

```diff
@@ -183,6 +188,6 @@ def run(
             logger.error(f"Face Upscale failed: {e}")
             face_helper = None
             del face_helper
-            torch.cuda.empty_cache()
+            safe_cuda_cache_empty()
```

Comment:
How can pytorch not be available here?

Reply:
It's not about pytorch not being available. Someone is having an issue currently where they get a CUDA misalignment issue from the cache clearing calls, so these are all here to make sure that happening doesn't stop execution.

However, it doesn't seem to have helped since their CUDA gets locked up.... Even completely removing the clearing didn't fix that, so i think their system is just f'd up somehow